app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from os import rename, mkdir, remove, path, getcwd
```

Comment:
You could import only the `os` module and call all required methods from it, e.g. `os.remove`, etc.
```suggestion
import os
```